engine/runtime/src/main/java/org/enso/interpreter/node/callable/FunctionCallInstrumentationNode.java

```diff
@@ -173,8 +173,15 @@ public boolean hasTag(Class<? extends Tag> tag) {
   /** @return the source section of this node. */
   @Override
   public SourceSection getSourceSection() {
-    Node parent = getParent();
-    return parent == null ? null : parent.getSourceSection();
+    var parent = getParent();
+    while (parent != null) {
+      var ss = parent.getSourceSection();
```

Comment:
Fixes an `assert` that: if `RootNode` has `SourceSection` any `Node` instrumented by `StandardsTag` has to have a `SourceSection` as well.